docs/administration/project_organization/bundle_extension/create_bundle.md

```diff
@@ -0,0 +1,264 @@
+
+
+```bash
+php bin/ibexa-bundle-generator currency-exchange-rate currency-exchange-rate-dir  --vendor-name=acme --vendor-namespace=ACME --bundle-name=CurrencyExchangeRate  --skeleton-name=extension
+```
+
+## Create bundle with GitHub template
+
+1\. Go to the [[= product_name_base =]] [GitHub repository](https://github.com/ibexa/bundle-template).
+
+2\. In the upper-right corner, click the **Use this template** button, and select **Create a new repository**.
+
+3\. Provide repository name. Optionally, you can add description for the bundle.
+Next, click **Create repository**.
+
+![GitHub template](bundle_github_template.png)
+
+
+Once the repository is created, a workflow starts which generates a bundle structure.
+
+Vendor namespace is generated from the orgnization name.
+Package and bundle name inherits from repository name.
+
+
+### Bundle directory structure
+
+Generated bundle consists of the following structure:
+
+```
+.
+├── LICENSE
+├── README.md
+├── composer.json
+├── deptrac.yaml
+├── package.json
+├── phpstan.neon
+├── phpunit.xml.dist
+├── src
+│   ├── bundle
+│   │   ├── AcmeCurrencyExchangeRateBundle.php
+│   │   ├── DependencyInjection
+│   │   │   └── AcmeCurrencyExchangeRateExtension.php
+│   │   └── Resources
+│   │       ├── config
+│   │       │   ├── prepend.yaml
+│   │       │   ├── services
+│   │       │   └── services.yaml
+│   │       └── views
+│   │           └── themes
+│   │               ├── admin
+│   │               └── standard
+│   ├── contracts
+│   └── lib
+└── tests
+    ├── bundle
+    ├── integration
+    └── lib
+```
+Where:
+
+- `LICENSE` - a license file, GPL v2 by default.
+- `README.md` - a readme file with bundle description, its version and install instructions.
+- `composer.json` - a package definition.
+- `deptrac.yaml` - a tool for static code analysis for PHP, checks the coherence of package architecture, for more information see [deptrac](https://qossmic.github.io/deptrac/) documentation.
+- `package.json` - frontend dependencies, for more information, see [about packages and modules](https://docs.npmjs.com/about-packages-and-modules).
+- `phpstan.neon` - phpstan configuration, a tool for static code analysis for PHP, scans, and evaluates codebase to find errors, and bugs, for more information, see the [documentation](https://phpstan.org/user-guide/getting-started).
+- `phpunit.xml.dist` - config for phpunit, unit and integration tests - see [documentation](https://phpunit.de/getting-started/phpunit-10.html).
+- `src` and `tests` follow the base catalog structure, according to [package structure](https://phpunit.de/getting-started/phpunit-10.html) docs.
+
+To fully use the possibilities of the bundle, get familiar with the structure:
+
+- `resources/`
+    - `config/` - contains configuration for the environment/ governs budle configuration
+        - `services/` - recommended place for services definition files
+        all services definitions must be split into separate files
```

Comment:
```suggestion
        - `services/` - recommended place for services definition files, all services definitions must be split into separate files.
```
With a line wrap, it looked weird in the preview
<img width="779" alt="Screenshot 2023-11-17 at 08 39 08" src="https://github.com/ibexa/documentation-developer/assets/35031941/5483d13c-b184-4779-9390-e594958059a1">